src/run_docker.py

```diff
@@ -61,30 +59,11 @@ async def metrics_pre_launch_check() -> Optional[PreLaunchCheckResult]:
             logging_context.log(f'Monitoring disabled. Unable to acquire authorization token.')
             return None
 
-        fast_check_result = await metrics_initial_check(gcp_session, dt_session, token)
```

Comment:
All these comments are related. We had this fastcheck that had a conditional based on the deployment type. For Cloud Functions, it was checking connectivity with DT. For k8s, it was useless, since all those connectivity checks, checking available projects, etc., are being done in the main loop. So I removed it.